apps/astarte_housekeeping/lib/astarte_housekeeping/migrator.ex

```diff
@@ -78,7 +78,7 @@ defmodule Astarte.Housekeeping.Migrator do
     query = """
     SELECT table_name
     FROM system_schema.tables
-    WHERE keyspace_name='astarte' AND table_name='kv_store'
+    WHERE keyspace_name='#{Config.astarte_instance_id!()}astarte' AND table_name='kv_store'
```

Comment:
This should be computed using ` CQLUtils.realm_name_to_keyspace_name("astarte", Config.astarte_instance_id!())` rather than just interpolating the instance id (also in the lines below)